contrib/ctr-remote/commands/rpull.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/containerd/containerd/images"
 	"github.com/containerd/containerd/labels"
 	"github.com/containerd/containerd/log"
-	"github.com/dragonflyoss/image-service/contrib/nydus-snapshotter/pkg/label"
+	"github.com/containerd/nydus-snapshotter/pkg/label"
```

Comment:
snapshotter has been relocated for quite a while. How the build succeed with the wrongly imported package url?

Reply:
An old committed package version is defined in `go.sum`.